front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/ApplicationsController.groovy

```diff
@@ -107,10 +115,12 @@ public class ApplicationsController {
   @RequestMapping(method = RequestMethod.POST)
   Application create(@RequestBody final Application app) {
     Application createdApplication = getApplication().initialize(app).withName(app.getName()).save()
-    try {
-      fiatService.ifPresent { it.sync() }
-    } catch (Exception ignored) {
-      log.warn("failed to trigger fiat permission sync", ignored)
+    if (fiatClientConfigurationProperties.isEnabled() && fiatConfigurationProperties.getRoleSync().isEnabled() && fiatService.isPresent()) {
```

Comment:
I think you want to wire in the FiatStatus component here instead and check isEnabled on that, that is used elsewhere as the determiner that fiat is enabled (it is just a wrapper around FiatClientConfigurationProperties that supports toggling the enabled status at runtime)

Reply:
Thank you, sounds good! Yeah, was not sure if there was a reason we were using `FiatStatus` in Gate but not Front50, but am happy to wire that in to this service (can also replace the existing checks with that in a separate commit for consistency, as I think there are only a handful of places to update). Will ping for a final look once that is complete. Thanks again for pointing that out! 🙏

Reply:
Added two additional commits: one to wire a `FiatStatus` bean, and one to consume it in the `ApplicationsController`. I didn't end up refactoring the other consumers of `FiatClientConfigurationProperties.isEnabled` because we might want to end up cherry-picking this to help debug the release branch tests as well, and wanted to keep this fix as minimal as possible.